docs/releases/2.2.txt

```diff
@@ -189,6 +189,9 @@ Models
   :meth:`.QuerySet.bulk_create` to ``True`` tells the database to ignore
   failure to insert rows that fail uniqueness constraints or other checks.
 
+* :class:`~django.db.models.FilePathField` ``path`` parameter now accepts a
```

Comment:
I only see the modification in `forms/fields.py`, not `models/fields`. Am I missing something here?